modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -397,7 +355,7 @@ Diagram.propTypes = {
 			fill: PropTypes.string,
 			id: PropTypes.number,
 			label: PropTypes.string,
-			linked_to_sku: PropTypes.oneOf(['sku', 'diagram']),
+			linkedToSku: PropTypes.oneOf(['sku', 'diagram']),
```

Comment:
We should also move `'diagram'` to a constant

```suggestion
			linkedToSku: PropTypes.oneOf([SKU, DIAGRAM]),
```